src/ai/backend/web/proxy.py

```diff
@@ -206,11 +207,11 @@ async def web_handler(request: web.Request, *, is_anonymous=False) -> web.Stream
             if proxy_path == "pipeline":
                 aiohttp_session = request.cookies.get("AIOHTTP_SESSION")
                 if not (sso_token := request.headers.get("X-BackendAI-SSO")):
-                    jwt_secret = request.app["config"]["pipeline"]["jwt"]["secret"]
-                    now = datetime.now(tz=timezone(timedelta(hours=9)))
+                    jwt_secret = config["pipeline"]["jwt"]["secret"]
+                    now = datetime.now().astimezone()
                     payload = {
                         # Registered claims
-                        "exp": now + timedelta(hours=1),
+                        "exp": now + timedelta(seconds=config["session"]["max_age"]),
```

Comment:
Same here.

Reply:
This one already has corresponding one: https://github.com/lablup/backend.ai/blob/9c512d54337aa53eb20f09c198bbffa179c71b57/src/ai/backend/web/config.py#L112-L122